src/main/webapp/WEB-INF/tags/instructor/course/courseEditCourseInfo.tag

```diff
@@ -49,6 +49,21 @@
             </div>
             
             <div class="form-group">
+                <label class="col-sm-3 control-label">Time Zone:</label>
+                <div class="col-sm-6">
+                    <select class="form-control"
+                            name="<%=Const.ParamsNames.COURSE_TIME_ZONE%>" id="<%=Const.ParamsNames.COURSE_TIME_ZONE%>"
+                            data-toggle="tooltip" data-placement="top" title="The time zone for the course, e.g. Asia/Singapore."
+                            tabindex="3" disabled>
+                        <option value="">Select a time zone...</option>
+                    </select>
+                </div>
+                <div class="col-sm-1">
+                    <input class="btn btn-primary" id="auto-detect-timezone" disabled value="Auto-Detect Timezone">
```

Comment:
Also to standardize things, perhaps add a space in `Timezone` in the value, as well as a hyphen in `timezone` in the id